src/test/regress/sql/alter_database_propagation.sql

```diff
@@ -56,4 +48,64 @@ alter database regression set lock_timeout from current;
 alter database regression set lock_timeout to DEFAULT;
 alter database regression RESET lock_timeout;
 
+set citus.enable_create_database_propagation=on;
+create database regression2;
+alter database regression2 with CONNECTION LIMIT 100;
+alter database regression2 with IS_TEMPLATE true CONNECTION LIMIT 50;
+alter database regression2 with IS_TEMPLATE false;
```

Comment:
Can we add a test for any of those three options where dbname needs quoting, such as "regression!\'2"?

Reply:
changed regression2 to "regression!'2"